html_css/intermediate_html/form-basics.md

```diff
@@ -415,7 +415,8 @@ To create a fieldset, we use the `<fieldset>` element. Whatever form inputs we w
 
 **Legend**
 
-The legend element is used to give a fieldset a heading or caption so the user can see what a grouping of inputs is for.
+
```

Comment:
Looks like a line was accidentally added when you fixed the conflict. If you could take that out, we'll be done!

```suggestion
```